src/main/java/edu/harvard/iq/dataverse/DatasetVersion.java

```diff
@@ -944,9 +944,29 @@ public String getReturnToDatasetURL(String serverName, Dataset dset) {
             }
         }
         return serverName + "/dataset.xhtml?id=" + dset.getId() + "&versionId" + this.getId();
+    } 
+    
+    /*
+    Per #3511 we  are returning all users to the File Landing page
+    If we in the future we are going to return them to the referring page we will need the 
+    getReturnToDatasetURL method and add something to the call to the api to
+    pass the referring page and some kind of decision point in  the getWorldMapDatafileInfo method in 
+    WorldMapRelatedData
+    SEK 3/24/2017
+    */
+    
+    public String getReturnToFilePageURL (String serverName, Dataset dset, DataFile dataFile){
```

Comment:
@sekmiller I see you added this new method but didn't delete the old one (getReturnToDatasetURL). Should we delete the old one? It isn't being used anymore. (I did a "Find Usages".)